code/modules/jobs/job_types/civilian.dm

```diff
@@ -13,6 +13,7 @@ Strip out?
 	spawn_positions = 1
 	supervisors = "the executive officer"
 	selection_color = "#dddddd"
+	alt_titles = list("Morale Officer")
```

Comment:
Hm.. this is kinda memey as an officer role, how about Morale Technician or something? Again give the clown an inch and he'll take a mile. Tell the clown he's an officer and all of a sudden unspeakable things happen.